vhdbuilder/packer/install-dependencies.sh

```diff
@@ -142,11 +142,10 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
   echo "  - [installed] containerd v${containerd_version}-${containerd_patch_version}" >> ${VHD_LOGS_FILEPATH}
 
   CRICTL_VERSIONS="
```

Comment:
guess it would be nice to eventually switch to use the json too heh. only in the test rn I think?

Reply:
Yes, it would be good to change here too